standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1752,13 +1763,28 @@ struct ExtendedTableInfo {
  4: optional list<string> requiredWriteCapabilities // capabilities required for write access
 }
 
+struct DropTableRequest {
+ 1: optional string catalogName,
+ 2: required string dbName,
```

Comment:
Some args should be required, such as the dbName, tblName

Reply:
dbname and tablename are already marked as required fields here.